sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisErrorSuite.scala

```diff
@@ -164,7 +164,7 @@ class AnalysisErrorSuite extends AnalysisTest {
 
   errorTest(
     "nested aggregate functions",
-    testRelation.groupBy('a)(
+    testRelation.groupBy()(
```

Comment:
The previous version caused some exceptions here though, I checked again and I found the current version caused nothing. So, I'll revert this.